src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -20,6 +20,25 @@
+		0138F53625ECD2D700A9B075 /* check_in.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F53225ECD2D700A9B075 /* check_in.pb.swift */; };
+		0138F53725ECD2D700A9B075 /* event.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F53325ECD2D700A9B075 /* event.pb.swift */; };
+		0138F54825ECD44300A9B075 /* app_features.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F53C25ECD44300A9B075 /* app_features.pb.swift */; };
+		0138F54925ECD44300A9B075 /* ppdd_edus_parameters.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F53D25ECD44300A9B075 /* ppdd_edus_parameters.pb.swift */; };
+		0138F54A25ECD44300A9B075 /* ppdd_ppac_parameters.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F53E25ECD44300A9B075 /* ppdd_ppac_parameters.pb.swift */; };
+		0138F54B25ECD44300A9B075 /* semantic_version.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F53F25ECD44300A9B075 /* semantic_version.pb.swift */; };
+		0138F54C25ECD44300A9B075 /* ppdd_ppa_parameters.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F54025ECD44300A9B075 /* ppdd_ppa_parameters.pb.swift */; };
+		0138F54D25ECD44300A9B075 /* app_config_ios.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F54125ECD44300A9B075 /* app_config_ios.pb.swift */; };
+		0138F54E25ECD44300A9B075 /* key_download_parameters.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F54225ECD44300A9B075 /* key_download_parameters.pb.swift */; };
+		0138F54F25ECD44300A9B075 /* ppdd_els_parameters.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F54325ECD44300A9B075 /* ppdd_els_parameters.pb.swift */; };
+		0138F55025ECD44300A9B075 /* exposure_detection_parameters.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F54425ECD44300A9B075 /* exposure_detection_parameters.pb.swift */; };
+		0138F55125ECD44300A9B075 /* risk_calculation_parameters.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F54525ECD44300A9B075 /* risk_calculation_parameters.pb.swift */; };
+		0138F55225ECD44300A9B075 /* app_config_android.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F54625ECD44300A9B075 /* app_config_android.pb.swift */; };
+		0138F55325ECD44300A9B075 /* event_registration_parameters.pb.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F54725ECD44300A9B075 /* event_registration_parameters.pb.swift */; };
+		0138F56125ECDC7700A9B075 /* TransmissionRiskValueMapping.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0138F56025ECDC7700A9B075 /* TransmissionRiskValueMapping.swift */; };
+		0138F5B425ED3B8800A9B075 /* default_app_config_115 in Resources */ = {isa = PBXBuildFile; fileRef = 0138F5B325ED3B8800A9B075 /* default_app_config_115 */; };
+		0138F5B525ED3B8800A9B075 /* default_app_config_115 in Resources */ = {isa = PBXBuildFile; fileRef = 0138F5B325ED3B8800A9B075 /* default_app_config_115 */; };
```

Comment:
twice?

Reply:
I guess its listed twice because it is member of the ENA and the ENATests target